### PR TITLE
[TOOLS-2527] limitInDays changed to Integer

### PR DESCRIPTION
**Pull request by herbert-venancio**
Reduced regular expression overuse by pre-converting string into integer representation

### CODE REVIEW

File: core/src/main/java/objective/taskboard/domain/converter/CardVisibilityEvalService.java [hunk 1]
```diff
@@ -34,31 +37,29 @@ public boolean isStillInVisibleRange(Long status, Instant lastUpdatedDate, List<
...
-        List<Filter> filters = filterRepository.getCache();
+        List<IssuesConfiguration> filters = laneService.getFilters();
         if (filters.isEmpty())
             return Optional.empty();
-        
-        Optional<Filter> filter = filters.stream()
-                .filter(f->f.getStatusId() == status)
+
+        Optional<IssuesConfiguration> filter = filters.stream()
+                .filter(f->f.getStatus() == status)
```
Comment (jytakano) [inline core/src/main/java/objective/taskboard/domain/converter/CardVisibilityEvalService.java]: Filtrar por issue type também

File: core/src/main/java/objective/taskboard/data/IssuesConfiguration.java [hunk 2]
```diff
@@ -1,45 +1,49 @@
...
+                .map(days -> valueOf(days.replaceAll("[^0-9-]", "")))
+                .orElse(null);
+        return new IssuesConfiguration(filter.getIssueTypeId(), filter.getStatusId(), limitInDays);
     }
 
     public boolean matches(Issue issue) {
         return issue.getType() == getIssueType() && issue.getStatus() == getStatus();
     }
+
+    public boolean matches(JiraIssueDto issue) {
```
Comment (jytakano) [inline core/src/main/java/objective/taskboard/data/IssuesConfiguration.java]: Not used

Comment (herbert-venancio) [reply core/src/main/java/objective/taskboard/data/IssuesConfiguration.java]: This is part of refactoring, the other pull request related to the same issue

Comment (jytakano) [reply core/src/main/java/objective/taskboard/data/IssuesConfiguration.java]: It is not used in the other pull request too

**Pull request merged**